src/Pages/Templates/TemplateDetails/components/TemplateDetailsTabs.tsx

```diff
@@ -22,72 +20,74 @@ export default function TemplateDetailsTabs() {
 
   const [topTabKey, setTopTabKey] = useState<ContentTabType>(CONTENT_ROUTE);
   const [contentTabKey, setContentTabKey] = useState<ContentSubTabType>(PACKAGES_ROUTE);
-  const [systemsTabKey, setSystemsTabKey] = useState<string>('other');
+
+  //   TODO: Uncomment to use systems tabs
```

Comment:
there are some commented out things in here that could probably be removed now

Reply:
Yes, I kind of left them if we ever want to have sub-tabs for systems. I can remove these.